src/yoroi-wallets/cardano/shelley/constants.ts

```diff
@@ -0,0 +1,146 @@
+import {DefaultAsset, TokenInfo} from '../../types'
+export * from '../../cardano/constants'
+
+export const NETWORK_ID = 1 // mainnet
+export const WALLET_IMPLEMENTATION_ID = 'haskell-shelley'
+export const PURPOSE = 2147485500 // cip1852
+export const COIN_TYPE = 2147485463
+export const ACCOUNT_INDEX = 0
+export const HARD_DERIVATION_START = 2147483648
+export const CHIMERIC_ACCOUNT = 2
+export const STAKING_KEY_INDEX = 0
+export const STAKING_KEY_PATH = [
+  PURPOSE,
+  COIN_TYPE,
+  ACCOUNT_INDEX + HARD_DERIVATION_START,
+  CHIMERIC_ACCOUNT,
+  STAKING_KEY_INDEX,
+]
+export const CHAIN_NETWORK_ID = 1
+export const BIP44_DERIVATION_LEVELS = {
+  ROOT: 0,
+  PURPOSE: 1,
+  COIN_TYPE: 2,
+  ACCOUNT: 3,
+  CHAIN: 4,
+  ADDRESS: 5,
+} as const
+export const REWARD_ADDRESS_ADDRESSING = {
+  path: [PURPOSE, COIN_TYPE, ACCOUNT_INDEX + HARD_DERIVATION_START, CHIMERIC_ACCOUNT, STAKING_KEY_INDEX],
+  startLevel: BIP44_DERIVATION_LEVELS.PURPOSE,
```

Comment:
startLevel for reward address on BIP44_PURPOSE 👀 @lisicky

Reply:
before:
```
private getRewardAddressAddressing() {
    return {
      path: [
        this.getPurpose(),
        CONFIG.NUMBERS.COIN_TYPES.CARDANO,
        CONFIG.NUMBERS.ACCOUNT_INDEX + CONFIG.NUMBERS.HARD_DERIVATION_START,
        CONFIG.NUMBERS.CHAIN_DERIVATIONS.CHIMERIC_ACCOUNT,
        CONFIG.NUMBERS.STAKING_KEY_INDEX,
      ],
      startLevel: CONFIG.NUMBERS.BIP44_DERIVATION_LEVELS.PURPOSE,
    }
  }
```